src/Microsoft.Diagnostics.Monitoring.Extension.Common/EgressHelper.cs

```diff
@@ -83,13 +74,68 @@ internal sealed class EgressHelper
                 result.FailureMessage = ex.Message;
             }
 
+            return ProcessEgressResult(result);
+        }
+
+        private static async Task<int> Validate<TProvider, TOptions>(Action<IServiceCollection> configureServices, CancellationToken token)
+            where TProvider : EgressProvider<TOptions>
+            where TOptions : class, new()
+        {
+            EgressArtifactResult result = new();
+            try
+            {
+                ExtensionEgressPayload configPayload = await GetPayload(token);
+                await using var serviceProvider = BuildServiceProvider<TProvider, TOptions>(configureServices, configPayload);
+
+                TOptions options = serviceProvider.GetRequiredService<IOptionsSnapshot<TOptions>>().Get(configPayload.ProviderName);
+
+                result.ArtifactPath = string.Empty;
+                result.Succeeded = true;
+            }
+            catch (Exception ex)
```

Comment:
Only catch OptionsValidationException; all other exceptions should flow through to help indicate some error in the system.